lib/Weasel/Widgets/Dojo/Option.pm

```diff
@@ -31,11 +31,26 @@ sub click {
     my ($self) = @_;
     my $popup = $self->_option_popup;
 
+    my $id = $popup->get_attribute('dijitpopupparent');
+    my $selector = $self->find("//*[\@id='$id']");
     if (! $popup->is_displayed) {
-        my $id = $popup->get_attribute('dijitpopupparent');
-        $self->find("//*[\@id='$id']")->click; # pop up the selector
+        $selector->click; # pop up the selector
+        $self->session->wait_for(
+          # Wait till popup opens
+          sub {
+            my $class = $selector->get_attribute('class');
+            return scalar( grep { $_ eq 'dijitHasDropDownOpen' }
```

Comment:
`scalar(...)` is true when it doesn't equal to zero. meaning that `scalar(...) != 0` and `scalar(...)` are equivalent. Elsewhere in the code (and in LedgerSMB), we use the latter.